### ISSUE TITLE
Bake mobile scrolling into skrollr itself

### ISSUE THREAD

**Issue by Prinzhorn** [Prinzhorn/skrollr#182]
I'm thinking about dropping iScroll for multiple reasons and instead implement a simple scrolling and put it into skrollr itself. Here's why

We are currently setting `useTransform:false` when instantiating iScroll, because using transform would break `position:fixed`. This causes poor performance, because scrolling is done using `top` and `position:absolute`. This could possibly be solved by putting fixed elements in a different container and only transform the other container. iScroll though doesn't support this, because it always uses the passed DOM element and it's first children.

Some key points:
- No desktop support (no mouse, draggable scrollbars or anything fancy).
- Only for body-scroll. We don't need scrolling inside an arbitrary DIV. Just listen to touch events on the `document`
- Just vertical scrolling. No zooming, pull to refresh or anything that skrollr does not need.
- In short: The minimal possible code which looks like native scrolling but overcomes the native limitations.
